src/components/BecomeAProvider.svelte

```diff
@@ -10,13 +10,19 @@
   import RequestToBeProvider from '$components/RequestToBeProvider.svelte';
 
   let hasRequestedToBeProvider = $state(false);
+  let newUserInitialVal = $user?.address === '' ? null : $user;
```

Comment:
The default value is `{address: '', isProvider: false}`. To maintain other logic throughout the app, and display the unselected state in the dropdown, I added this line.